release-notes/major10/10.5.0/migration.md

```diff
@@ -0,0 +1,28 @@
+# Migration
+
+This page describes how to update Valtimo from the previous version to the current.
+
+* **Migrate from process 'Retrieve Document and link to Zaak' to process 'Link document to zaak'**
+
+  Scope: front-end
+
+    1. **Step1**
```

Comment:
Either change this to a title that means something, or remove the text and instead to something like:

1. Under the 'Admin' menu, go to the 'Process links' menu.

This should be done for everything here.